GGProject/static/admin/css/fonts.css

```diff
@@ -0,0 +1,20 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('../fonts/Roboto-Bold-webfont.woff');
+    font-weight: 700;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto';
+    src: url('../fonts/Roboto-Regular-webfont.woff');
+    font-weight: 400;
+    font-style: normal;
+}
+
```

Comment:
Looks like there are two `@font-face {`, can we combine them?

Reply:
It downloads multiple fonts so we should keep them separate.